pycbc/detector.py

```diff
@@ -322,3 +304,96 @@ def overhead_antenna_pattern(right_ascension, declination, polarization):
 def effective_distance(distance, inclination, f_plus, f_cross):
     return distance / np.sqrt( ( 1 + np.cos( inclination )**2 )**2 / 4 * f_plus**2 + np.cos( inclination )**2 * f_cross**2 )
 
+    """ LISA class  """
+
+class LISA(object):
+    def __init__(self, kappa, _lambda_, reference_time = 1126259462.0):
+        self.reference_time = reference_time
+        self.kappa = kappa
+        self._lambda_ = _lambda_
+
+    def get_pos(self, t_gps):
+        if t_gps is None:
+            t_gps = Time(val = self.reference_time, format = 'gps',
+                         scale = 'utc').to_datetime(timezone = None)
+        elif isinstance(t_gps, np.ScalarType):
+            t_gps = Time(val = t_gps, format = 'gps',
+                         scale = 'utc').to_datetime(timezone = None)
+
+        t_gps = np.sum(np.array([t_gps.year - 2034, t_gps.month/12,
```

Comment:
use the astropy Time class directly to convert to gps time (it will be more generic).